benchmark/swebench-act-context/upstream/edit_patch_v2.py

```
@@ -0,0 +1,152 @@
+    return out
+
+
+def _find_block(content: str, search: str) -> tuple[int, int, int]:
+    """Locate `search` in `content`. Returns (n_matches, start, end) for the first match.
+    Exact match first, then trailing-whitespace-tolerant line match."""
+    n = content.count(search)
+    if n:
+        i = content.index(search)
+        return n, i, i + len(search)
+    clines = content.split("\n")
+    slines = [l.rstrip() for l in search.split("\n")]
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)